.circleci/config.yml

```diff
@@ -4,8 +4,8 @@ parameters:
   workflow:
     description: Which workflow to run
     type: enum
-    enum: ['ci', 'pr', 'merged', 'daily']
-    default: 'ci'
+    enum: ['noop', 'pr', 'merged', 'daily']
+    default: 'noop'
```

Comment:
I guess this is the simplest change, but I wonder if there's a way to disable CircleCI triggering automatically at all. We always trigger CircleCI via GitHub Actions that calls the API, so there's really no reason for CircleCI to ever listen for changes automatically. I could be wrong here, maybe something like `pretty-docs` depends on it or something.

Reply:
I removed the `noop` workflow. Now I try to figure out, what else causes CircleCI to run